recipe/meta.yaml

```diff
@@ -7,24 +7,34 @@ package:
 
 source:
   - url: https://github.com/llvm/llvm-project/releases/download/llvmorg-{{ version.replace(".rc", "-rc") }}/llvm-project-{{ version.replace(".rc", "rc") }}.src.tar.xz
-    sha256: 9c4278a6b8884eb7f4ae7dfe3c8e5445019824885e47cfdf1392563c47316fd6
+    sha256: 9d53ad04dc60cb7b30e810faf64c5ab8157dadef46c8766f67f286238256ff92
     patches:
       # https://reviews.llvm.org/D99470
       - 0001-Support-cross-compiling-standalone-MLIR.patch
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   build:
     - cmake
     - ninja
     - {{ compiler('cxx') }}
     - llvmdev {{ version }}  # [build_platform != target_platform]
+    {% if not arm64 %}
+    - python                 # [build_platform != target_platform]
+    - numpy >=1.23           # [build_platform != target_platform]
+    - pybind11 >=2.8.0       # [build_platform != target_platform]
```

Comment:
Please remove these from every section except the python bindings. Otherwise all the outputs will get a matrix of builds per python version.

Reply:
Should they stay in the main requirements/build section? Or is the only mention of python in the outputs/mlir-python-bindings/requirements/build section?

Reply:
How do I get a matrix of builds per python version only for the mlir-python-bindings output? Is that possible or has this exceeded the complexity of what a single feedstock can deliver?